### ISSUE TITLE
Digitization in the form of collection inventories

### ISSUE THREAD

**Issue by ekrimmel** [paleo-data/happy-hours#31]
## What is your idea for a happy hour topic?
Collections may maintain data at an organizational level higher than specimens, e.g. inventory data about collecting localities. These data may need to be managed in a separate CMS, and or shared differently from specimen records. Topic could include discussing Latimer Core.

## Who could be involved in leading this happy hour?
_[add text here - feel free to suggest yourself or others]_

## Are there any timing considerations?
_[add text here - e.g. needs to happen before xyz conference, after field season, ASAP, etc.]_

## Next steps
This issue captures a happy hour topic idea. Once you are ready to finalize an idea and put it on the PDWG happy hour schedule, please create a new issue using the [Plan a happy hour](https://github.com/paleo-data/happy-hours/issues/new?assignees=&labels=PDWG&projects=&template=plan-happy-hour.md&title=%5Breplace+this+text+with+the+happy+hour+topic%5D) template. **Then, close this issue.**

**Comment by ekrimmel** [comment]
This came up at GBIF meeting in April 2024 and we decided that it might be a good topic for Fall 2024.

**Comment by ekrimmel** [comment]
This topic connections to #25 !

**Comment by ekrimmel** [comment]
scheduled for 2025-02-13
